packages/atomic/src/components/atomic-html/atomic-html.tsx

```diff
@@ -0,0 +1,46 @@
+import {Component, Prop, State, h} from '@stencil/core';
+import DOMPurify from 'dompurify';
+import {
+  InitializableComponent,
+  Bindings,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+
+/**
+ * The `atomic-html` component renders the HTML value of a string.
+ *
+ * There is an inherent XSS security concern associated with the usage of this component.
+ *
+ * Use only with values for which you are certain the content is harmless.
+ */
+@Component({
+  tag: 'atomic-html',
+  shadow: true,
+})
+export class AtomicHtml implements InitializableComponent {
+  @InitializeBindings() public bindings!: Bindings;
+  @State() public error!: Error;
+
+  /**
+   * The string value containg HTML to display;
+   */
+  @Prop({reflect: false}) public value!: string;
+  /**
+   * Specify if the content should be sanitized, using `DOMPurify` (https://www.npmjs.com/package/dompurify).
+   */
+  @Prop({reflect: true}) public sanitize = true;
+
+  public connectedCallback() {
+    if (!this.value) {
+      this.error = new Error('The "value" attribute must be defined.');
+    }
+  }
+
+  public render() {
+    return (
+      <span
+        innerHTML={this.sanitize ? DOMPurify.sanitize(this.value) : this.value}
```

Comment:
In which cases would a user not want to sanitize their html?

Reply:
If they want to display a source they really trust "as is". In the scenario that DOMPurify would improperly sanitize something that should not have been.